<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="5" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-03-28T14:45:07 IST" name="Suite" finished-at="2023-03-28T14:46:43 IST" duration-ms="95848">
    <groups>
      <group name="errorHandling">
        <method signature="ErrorValidationsTest.LoginErrorValidation()[pri:0, instance:eCommerce.Tests.ErrorValidationsTest@386f0da3]" name="LoginErrorValidation" class="eCommerce.Tests.ErrorValidationsTest"/>
      </group> <!-- errorHandling -->
    </groups>
    <test started-at="2023-03-28T14:45:07 IST" name="SubmitOrderTest" finished-at="2023-03-28T14:46:17 IST" duration-ms="70016">
      <class name="eCommerce.Tests.SubmitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:eCommerce.Tests.SubmitOrderTest@66c92293]" started-at="2023-03-28T14:45:07 IST" name="launchApplication" finished-at="2023-03-28T14:45:39 IST" duration-ms="31809" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="SubmitOrder(java.util.HashMap)[pri:0, instance:eCommerce.Tests.SubmitOrderTest@66c92293]" started-at="2023-03-28T14:45:39 IST" name="SubmitOrder" data-provider="getData" finished-at="2023-03-28T14:45:50 IST" duration-ms="10382" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Sdet@2023, product=ADIDAS ORIGINAL, email=sailesh.ramesh@sdet.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubmitOrder -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:eCommerce.Tests.SubmitOrderTest@66c92293]" started-at="2023-03-28T14:45:50 IST" name="TearDown" finished-at="2023-03-28T14:45:50 IST" duration-ms="316" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:eCommerce.Tests.SubmitOrderTest@66c92293]" started-at="2023-03-28T14:45:50 IST" name="launchApplication" finished-at="2023-03-28T14:46:01 IST" duration-ms="11200" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="SubmitOrder(java.util.HashMap)[pri:0, instance:eCommerce.Tests.SubmitOrderTest@66c92293]" started-at="2023-03-28T14:46:01 IST" name="SubmitOrder" data-provider="getData" finished-at="2023-03-28T14:46:06 IST" duration-ms="4865" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Iamking@000, product=ADIDAS ORIGINAL, email=shetty@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SubmitOrder -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:eCommerce.Tests.SubmitOrderTest@66c92293]" started-at="2023-03-28T14:46:06 IST" name="TearDown" finished-at="2023-03-28T14:46:06 IST" duration-ms="105" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:eCommerce.Tests.SubmitOrderTest@66c92293]" started-at="2023-03-28T14:46:06 IST" name="launchApplication" finished-at="2023-03-28T14:46:14 IST" duration-ms="7329" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="orderHistory()[pri:0, instance:eCommerce.Tests.SubmitOrderTest@66c92293]" started-at="2023-03-28T14:46:14 IST" name="orderHistory" finished-at="2023-03-28T14:46:16 IST" depends-on-methods="eCommerce.Tests.SubmitOrderTest.SubmitOrder" duration-ms="2935" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderHistory -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:eCommerce.Tests.SubmitOrderTest@66c92293]" started-at="2023-03-28T14:46:16 IST" name="TearDown" finished-at="2023-03-28T14:46:17 IST" duration-ms="280" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- eCommerce.Tests.SubmitOrderTest -->
    </test> <!-- SubmitOrderTest -->
    <test started-at="2023-03-28T14:46:18 IST" name="ErrorValidationTest" finished-at="2023-03-28T14:46:43 IST" duration-ms="24636">
      <class name="eCommerce.Tests.ErrorValidationsTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:eCommerce.Tests.ErrorValidationsTest@386f0da3]" started-at="2023-03-28T14:46:18 IST" name="launchApplication" finished-at="2023-03-28T14:46:28 IST" duration-ms="10406" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:eCommerce.Tests.ErrorValidationsTest@386f0da3]" started-at="2023-03-28T14:46:28 IST" name="LoginErrorValidation" finished-at="2023-03-28T14:46:29 IST" duration-ms="976" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password] but found [Incorrect email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password] but found [Incorrect email or password.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at eCommerce.Tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:eCommerce.Tests.ErrorValidationsTest@386f0da3]" started-at="2023-03-28T14:46:29 IST" name="TearDown" finished-at="2023-03-28T14:46:30 IST" duration-ms="287" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:eCommerce.Tests.ErrorValidationsTest@386f0da3]" started-at="2023-03-28T14:46:30 IST" name="launchApplication" finished-at="2023-03-28T14:46:39 IST" duration-ms="9352" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:eCommerce.Tests.ErrorValidationsTest@386f0da3]" started-at="2023-03-28T14:46:39 IST" name="LoginErrorValidation" finished-at="2023-03-28T14:46:40 IST" duration-ms="876" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password] but found [Incorrect email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password] but found [Incorrect email or password.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at eCommerce.Tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:eCommerce.Tests.ErrorValidationsTest@386f0da3]" started-at="2023-03-28T14:46:42 IST" name="TearDown" finished-at="2023-03-28T14:46:43 IST" duration-ms="523" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- eCommerce.Tests.ErrorValidationsTest -->
    </test> <!-- ErrorValidationTest -->
  </suite> <!-- Suite -->
</testng-results>
